### ISSUE TITLE
html=true needed to produce plain text in email.jsp pages

### ISSUE THREAD

**Issue by ghost** [makumba/makumba#200]
**Reported by @frederikhabils on 16 May 2003 22:33 UTC**
the html= tag param of mak:value dictates whether to HTML-escape the text/char
[] or not. 

html=true, means that the value contains HTML code, and should not be escaped. 
The default is html=false, as Makumba is (naturally) assuming that producing 
HTML pages is the default - HTML-escaping being necessary.

I've realised that the JSP pages that send mails with mailer taglib (e.g. 
http://private.best.eu.org/forgotPassMail.jspx), should not HTML-escape the 
values, because they are not viewed in a browser, but in a mail client. And 
the format of the mail is text. 

in other words, in the above situation html=true is needed, not because the 
value contains HTML code, but plain text.

For example, this is an extract of a sent mail, where html=true is NOT used.
  -- 
  Activity : &quot;Environmental Engineering&quot;
  --

I just wanted to point this out, because it means using html=true in a manner 
that seems contradictory to the one suggested by its name.

Apart from this being counter-intuitive, I can't really pinpoint any exact 
problem or solution :-)

Migrated-From: http://trac.makumba.org/ticket/440

**Comment by ghost** [comment]
**Comment by @cristianbogdan on 22 May 2003 14:12 UTC**
i think html= should be deprecated and escapeHtml= should be added, with true as
default value

**Comment by ghost** [comment]
**Comment by @frederikhabils on 30 Jun 2003 22:29 UTC**
0.6 (or 0.5.9.x) might be as good a time as any to introduce this escapeHtml ?

Although I got used to the idea already, I must admit raising an eyebrow at
first reading of "escapeHtml"... perhaps slightly too technical term? If we were
to care about that, "convertToHtml" could be considered more non-techie. Ideas?

**Comment by ghost** [comment]
**Modified by @manuelbernhardt on 28 Jun 2010 07:22 UTC**

**Comment by ghost** [comment]
**Modified by @manuelbernhardt on 28 Jun 2010 07:36 UTC**

**Comment by ghost** [comment]
**Comment by @cristianbogdan on 20 Mar 2013 09:56 UTC**
this is about a wrong attribute name, not a bug per se.
